### ISSUE TITLE
Add more boards

### ISSUE THREAD

**Issue by after-ephemera** [azjkjensen/sudoku-validator#2]
Add more boards to the boards/ directory, testing more edge cases. 

Entries must be comma-separated and rows must be newline (`\n`) separated.

**Comment by Anirudh257** [comment]
Hey, I would love to work on this one too. Do you need  more test cases( more number of valid sudoku board combinations) ?

**Comment by after-ephemera** [comment]
@Anirudh257 that's right. I'd like to have several more valid and invalid boards, named according to the type. One idea would be to maybe separate the `boards` directory into `validBoards` and `invalidBoards`.

**Comment by Anirudh257** [comment]
@azjkjensen I have made 2 pull requests(1 for valid board examples and other for invalid board examples). Please check if it is satisfies your request.

**Comment by Anirudh257** [comment]
I have made 1 test file containing both the valid and invalid sudoku boards
